parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -210,23 +210,31 @@ configureCNIIPTables() {
     fi
 }
 
+disableSystemdResolved() {
```

Comment:
I realize this name is misleading , it's not actually disablging the service but just the caching part, can you help rename it to disableSystemdResolvedCache? We still rely on it to sync the dns server list